src/ui/TypingIndicatorMessage/typingDots.scss

```diff
@@ -0,0 +1,66 @@
+@import '../../styles/variables';
+
+$dot-diameter-default: 8px;
+$duration: 1.4s;
+$blinked-scale: 1.2;
```

Comment:
![image](https://github.com/sendbird/sendbird-uikit-react/assets/26326015/8212b6ea-a3ef-40bc-924e-8fa37fd82a43)
It seems like the animation scale is set to 150% in Figma.
However, when I apply 1.5x, it looks too large 🤔 We should show the result to Gabie and double-check.

Reply:
yeah i will do that. That is why i set to 1.2

Reply:
Gabie says 1.2 is good.